utils/shipping-actions.js

```diff
@@ -154,3 +160,45 @@ export function selectOtherPickup() {
     .click()
   cy.get('.pkpmodal-details-confirm-btn').click()
 }
+
+export function fillShippingPreviewPickupAddress(account) {
+  cy.get('#find-pickup-link').click()
+
+  if (
+    [
+      ACCOUNT_NAMES.CLEAN_NO_MAPS,
+      ACCOUNT_NAMES.NO_LEAN,
+      ACCOUNT_NAMES.INVOICE,
+    ].some(localAccount => localAccount === account)
+  ) {
+    cy.waitAndGet('#pkpmodal-search #ship-postalCode', 3000).type('22071060')
+  } else {
+    cy.waitAndGet('#pkpmodal-search input', 3000).type('Praia de Botafogo, 300')
+
+    cy.get('.pac-item')
+      .first()
+      .trigger('mouseover', { force: true })
+
+    cy.get('.pac-item')
+      .first()
+      .click({ force: true })
+  }
+  cy.get('.pkpmodal-points-list .pkpmodal-pickup-point-main').click()
+
+  cy.get('.pkpmodal-details-confirm-btn').click()
+}
+
+export function checkShippingPreviewResult(selectors) {
+  selectors.forEach(selector => {
+    if (selector.name) {
+      cy.get('.srp-data')
+        .contains(selector.name)
+        .should('be.visible')
+    }
+    if (selector.text) {
+      cy.get('.srp-data')
+        .contains(selector.text)
+        .should('be.visible')
+    }
```

Comment:
This is not ideal because you might be matching `name` and `text` in different selectors. But this is fine for now.